api/userRoutes/users.js

```diff
@@ -16,11 +16,11 @@ router.get("/get-users", async (req, res, next) => {
 
 router.get("/get-following", async (req, res, next) => {
     try {
-        const user = await User.getUser();
-        const query = new Parse.Query("Follow");
-        query.equalTo("from", user);
-        const users = await query.find();
-        res.status(200).json({ users })
+        const currentUser = await User.getUser();
+        let followingPointers = currentUser.get("friends");
```

Comment:
should these be `const`?

Reply:
Just changed! and just rebased onto pr #35 to make sure there are no merge conflicts